android-components/components/feature/downloads/src/test/java/mozilla/components/feature/downloads/share/ShareDownloadFeatureTest.kt

```diff
@@ -141,39 +141,7 @@ class ShareDownloadFeatureTest {
         shareFeature.startSharing(shareState)
 
         verify(shareFeature).download(shareState)
-        verify(shareFeature).share(downloadedFile.canonicalPath, "contentType", null, "testUrl")
-        Unit
-    }
-
-    @Test
-    fun `startSharing() will not use a too long HTTP url as message`() = runTestOnMain {
-        val shareFeature = spy(ShareDownloadFeature(context, mock(), mock(), null))
-        val maxSizeUrl = "a".repeat(CHARACTERS_IN_SHARE_TEXT_LIMIT)
-        val tooLongUrl = maxSizeUrl + 'x'
-        val shareState = ShareInternetResourceState(url = tooLongUrl, contentType = "contentType")
-        val downloadedFile = File("filePath")
-        doReturn(downloadedFile).`when`(shareFeature).download(any())
-        shareFeature.scope = scope
-
-        shareFeature.startSharing(shareState)
-
-        verify(shareFeature).download(shareState)
-        verify(shareFeature).share(downloadedFile.canonicalPath, "contentType", null, maxSizeUrl)
-        Unit
-    }
-
-    @Test
-    fun `startSharing() will use an empty String as message for data URLs`() = runTestOnMain {
-        val shareFeature = spy(ShareDownloadFeature(context, mock(), mock(), null))
-        val shareState = ShareInternetResourceState(url = "data:image/png;base64,longstring", contentType = "contentType")
-        val downloadedFile = File("filePath")
-        doReturn(downloadedFile).`when`(shareFeature).download(any())
-        shareFeature.scope = scope
-
-        shareFeature.startSharing(shareState)
-
-        verify(shareFeature).download(shareState)
-        verify(shareFeature).share(downloadedFile.canonicalPath, "contentType", null, "")
+        verify(shareFeature).share(downloadedFile.canonicalPath, "contentType", null)
```

Comment:
nit: We could add a `null` check for the message instead of relying on the default value which might check without the test knowing.